### PR TITLE
AnshKanungo_logoutFunction

### PR DESCRIPTION
**Pull request by Anshkanungo**
Added LogOut Functionality, with Swal Pop-up Alert.

### CODE REVIEW

File: client/src/components/HomeNavbar/HomeNavbar.tsx [hunk 1]
```diff
@@ -102,6 +105,30 @@ const HomeNavbar = () => {
     );
   });
 
+  // A LogOut Function:-------
+  const navigate = useNavigate()
```
Comment (Gaurav-Verma07) [inline client/src/components/HomeNavbar/HomeNavbar.tsx]: Move this initialization to the top level inside the function (line 89).
A good code is when everything is placed at its specificed location.

File: client/src/components/HomeNavbar/HomeNavbar.tsx [hunk 2]
```diff
@@ -102,6 +105,30 @@ const HomeNavbar = () => {
...
+  function logOut(){
```
Comment (Gaurav-Verma07) [inline client/src/components/HomeNavbar/HomeNavbar.tsx]: - Use arrow function here.
- Use relevant function names, `logOutHandler`.
- Remove unnecessary comments.

**Comment by Anshkanungo** [comment]
I made the changes as per your instructions. plz verify.

**Comment by Gaurav-Verma07** [comment]
> I made the changes as per your instructions. plz verify.

Please use `window.confirm` instead of `sweetalert` .

**Comment by Anshkanungo** [comment]
Hey, I used sweetalert to add this type of visual effect to the logout Animation. Having a simple window's alert was not so striking visually. 
Should I keep it or revert it to window's alert? 
https://user-images.githubusercontent.com/56950431/219698562-d8b99309-c8c3-4c95-9e89-ebb3764a4d7c.mp4

**Comment by Gaurav-Verma07** [comment]
@Anshkanungo , I appreciate the way you utilize `sweetalert` package but the reason we are not accepting it because we are already using libraries like `mantine` that is quite heavy to load, and as per the project goal, the application will be utilized by student at various parts of the country having varied network speeds.
In this case, if we use more libraries then it'll greatly affect the speed of our website so our major goal is functionality not the UI but yeah in future if the need persists then we'll surely integrate it.
I hope it clears your query😊.

**Comment by Anshkanungo** [comment]
> @Anshkanungo , I appreciate the way you utilize `sweetalert` package but the reason we are not accepting it because we are already using libraries like `mantine` that is quite heavy to load, and as per the project goal, the application will be utilized by student at various parts of the country having varied network speeds. In this case, if we use more libraries then it'll greatly affect the speed of our website so our major goal is functionality not the UI but yeah in future if the need persists then we'll surely integrate it. I hope it clears your query😊.

I understand. I made the necessary changes. Pls see to it and let me know what more is needed.

**Pull request merged**